src/createComponent.js

```diff
@@ -16,6 +16,10 @@ import React from 'react';
  * @public
  */
 export default function createComponent(options = {}) {
+  if (!options.render || typeof options.render !== 'function') {
+    throw new Error('Render is a required method.');
```

Comment:
It would be even better to have the component's name in the error message:
`Component '${Component.name}'missing required method: 'render'`.  What do you think? cc/ @fahad19